src/odbc/src/statement.cpp

```diff
@@ -976,11 +989,11 @@ SqlResult::Type Statement::InternalFreeResources(int16_t option) {
 }
 
 void Statement::Close() {
-  IGNITE_ODBC_API_CALL(InternalClose());
+  IGNITE_ODBC_API_CALL(InternalClose(false));
 }
 
-SqlResult::Type Statement::InternalClose() {
-  if (!currentQuery.get())
+SqlResult::Type Statement::InternalClose(bool ignoreErrors) {
+  if (ignoreErrors || !currentQuery.get())
```

Comment:
This shouldn't return so early. We're missing calling currentQuery->Close() if this happens.

Reply:
Changed so that `currentQuery->Close()` will be called if `ignoreErrors` is passed in.